spec/vcr/applications_controller/show.yml

```diff
@@ -85,12 +85,12 @@ http_interactions:
       Cache-Control:
       - no-cache,must-revalidate,max-age=0,no-store,private
       Set-Cookie:
-      - BrowserId=a1NKcUKLQ0KnQiGoVFDdJQ;Path=/;Domain=.salesforce.com;Expires=Mon,
-        19-Nov-2018 18:24:27 GMT;Max-Age=5184000
+      - BrowserId=4g9nYTa4SP2CGLitfJInOQ;Path=/;Domain=.salesforce.com;Expires=Sun,
+        25-Nov-2018 15:42:38 GMT;Max-Age=5184000
       Expires:
       - Thu, 01 Jan 1970 00:00:00 GMT
       Sforce-Limit-Info:
```

Comment:
wow this is cool that we get this info back on a request. I wonder if that means we could track it externally somehow. cc @callaghanc

Reply:
yes, I saw that too. pretty cool

Reply:
@akegan for sure! anything we're getting in responses is potentially able to be used in logging or record-keeping. did you have some ideas about how you want to use this data? e.g. GA?